qt_ui/windows/QLiberationWindow.py

```diff
@@ -71,6 +71,7 @@ def __init__(self, game: Optional[Game]) -> None:
                     logging.info("Loading last saved game : " + str(last_save_file))
                     game = persistency.load_game(last_save_file)
                     self.onGameGenerated(game)
+                    self.updateWindowTitle(last_save_file)
```

Comment:
ope, I've accidentally assumed `last_save_file` was the same type as `file` (below), looks like it's not a tuple, so should be `self.updateWindowTitle((last_save_file, None)` instead?

Reply:
Other way around I think. The other callsite should be `self.update_window_title(file[0])`

I think you also need to update the name in `saveGameAs`, since that can change the name